### ISSUE TITLE
overLay bind click/dblclick but received the same event twice in listener for one click

### ISSUE THREAD

**Issue by heyouxian** [jsplumb/jsplumb#712]
"jsplumb": "^2.6.3"

rOverlay.bind("click", (value, originalEvent) => {
  console.log('clicked');
});

one mouse click but console print twice. And use "===" can judge the originalEvent is same for the twice invoking.

**Comment by sporritt** [comment]
do you have a jsfiddle or codepen?

**Comment by heyouxian** [comment]
no. How to use them? They can help to track event?

**Comment by sporritt** [comment]
i mean to show the problem.

**Comment by heyouxian** [comment]
I cannot reproduce this issue in https://codepen.io/heyouxian/pen/GQddZX, I don't know why because it's nearly same code and even I upgrade my local jsplumb to 2.6.7, the issue is still existing.
So I guess may because my frontend project is based on React? But from DOM tree I cannot see any redundant DOM element and from console.trace, the two times event trigger has totally same stack.

**Comment by sporritt** [comment]
are you sure the call to `bind` is only being executed once?

**Comment by szhllyyl** [comment]
fixed it？i met the same question 。maybe use unbind('click') fist? but it will be nice to tackle the problem at its source

**Comment by sporritt** [comment]
i dont think we actually managed to get to the source of the problem. is the bind call only being made once? we never found that out.
